docs/sources/metainfo/component.xml

```diff
@@ -567,6 +567,10 @@
 				<literal>recommends</literal> tag should be used. In this case, AppStream clients should allow the installation of the software
 				component, but may display a warning before allowing it.
 			</para>
+			<para>
+				To specify that a device is being used, an application can use the <literal>supports</literal> tag. This will mean that a certain requirement
```

Comment:
We never talk about "applications" in the generic component spec, because the component may be anything from a font to a shared library. The spec uses "component" or "software" (or "software component") exclusively instead.